test/Methods.test.ts

```diff
@@ -187,20 +193,36 @@ describe('test ES6 module', () => {
       expect(window.fdc3.addContextListener).toHaveBeenNthCalledWith(2, null, handler2);
     });
 
-    test('getSystemChannels should delegate to window.fdc3.getSystemChannels', async () => {
+    test('getUserChannels should delegate to window.fdc3.getUserChannels', async () => {
+      await getUserChannels();
+
+      expect(window.fdc3.getUserChannels).toHaveBeenCalledTimes(1);
+      expect(window.fdc3.getUserChannels).toHaveBeenCalledWith();
+    });
+
+    test('getSystemChannels should delegate to window.fdc3.getUserChannels', async () => {
       await getSystemChannels();
 
-      expect(window.fdc3.getSystemChannels).toHaveBeenCalledTimes(1);
-      expect(window.fdc3.getSystemChannels).toHaveBeenCalledWith();
+      expect(window.fdc3.getUserChannels).toHaveBeenCalledTimes(2); //was already called in previous test, hence this should be the second call to getUserChannels
```

Comment:
This makes the order of the tests significant and breaks their isolation from each other. You could reset the mocks in a beforeEach so that the call counts are always reset. https://jestjs.io/docs/jest-object#jestresetallmocks
```ts
  beforeEach(() => {
    jest.resetAllMocks();
  });
```

Reply:
good call! fixed.